aspnetcore/toc.md

```diff
@@ -15,14 +16,11 @@
 #### [With Visual Studio](xref:tutorials/first-mvc-app/index)
 #### [With Visual Studio Code](xref:tutorials/first-mvc-app-xplat/index)
 #### [With Visual Studio for Mac](xref:tutorials/first-mvc-app-mac/index)
-## Web APIs
-### [With Visual Studio](xref:tutorials/first-web-api)
-### [With Visual Studio Code](xref:tutorials/web-api-vsc)
-### [With Visual Studio for Mac](xref:tutorials/first-web-api-mac)
-### [For native mobile apps](xref:mobile/native-mobile-backend)
 ## Real-time web apps
 ### [SignalR with JavaScript](xref:tutorials/signalr)
 ### [SignalR with TypeScript](xref:tutorials/signalr-typescript-webpack)
+## [Create backend services for native mobile apps](xref:mobile/native-mobile-backend)
```

Comment:
I suggest removing the link from here, since this doc focuses on web API.

Reply:
I just moved that to the bottom or real-time. I'll let @tdykstra change the TOC